common/src/main/java/com/microsoft/identity/common/internal/providers/oauth2/AuthorizationStrategy.java

```diff
@@ -73,12 +74,12 @@ protected void launchIntent(@NonNull Intent intent) {
         final Fragment fragment = mReferencedFragment.get();
 
         if (fragment != null) {
-            final AuthorizationFragment authFragment = AuthorizationActivity.getAuthorizationFragmentFromStartIntent(intent);
+            final Fragment authFragment = AuthorizationActivityFactory.getAuthorizationFragmentFromStartIntent(intent);
```

Comment:
interesting...so this is no longer an `AuthorizationFragment`...should the method be renamed as well?

Reply:
I didn't want to disturb the existing base class and/or introduce another level in the hierarchy.  It does still return an authorization fragment... just not one that inherits from AuthorizationFragment ;-).  In other words... I think i'd prefer to address this when we adopt this as the new default behavior in the fall release.  I don't want to support both approaches long term.